src/server/server_family.cc

```diff
@@ -1622,10 +1643,13 @@ void ServerFamily::Info(CmdArgList args, ConnectionContext* cntx) {
     append("last_save", save_info->save_time);
     append("last_save_duration_sec", save_info->duration_sec);
     append("last_save_file", save_info->file_name);
+    size_t is_loading = service_.GetGlobalState() == GlobalState::LOADING;
+    append("loading", is_loading);
```

Comment:
Is there a way to test this in a simple way? I will need a large dataset to load and then during the loading phase I should issue an `INFO PERSISTENCE` command.

Reply:
Never worked with the rdb unit tests, but we have pytests that load a fairly large snapshot

Reply:
Cheers, will take a loo